prow/config.yaml

```diff
@@ -143,14 +143,24 @@ tide:
     - do-not-merge/work-in-progress
```

Comment:
you need to remove istio/istio.github.io from the previous query

Reply:
Done